platforms/Linux/centos/8/swift-lang.spec

```
@@ -0,0 +1,167 @@
+%global debug_package %{nil}
+%global swifttag 5.5-RELEASE
+%global swiftbuild swift-source
+%global icu_version 65-1
+%global yams_version 4.0.2
+%global sap_version 0.4.3
+%global swift_crypto_version 1.1.5
+
+Name:           swift-lang
+Version:        5.5
+Release:        1%{?dist}
+Summary:        Apple's Swift programming language
+License:        ASL 2.0 and Unicode
+URL:            https://swift.org
```

```suggestion
License:       Apache 2.0
URL:            https://swift.org
```

I think we should update the License to Apache 2.0.

---

We will have to do the same for Fedora spec file.